packages/dashboard/src/components/DashboardGridLayout/examples/DashboardGridLayout.Example.tsx

```diff
@@ -20,12 +21,19 @@ export class DashboardGridLayoutExample extends React.Component<{}, {}> {
     );
   }
 
+  private _onLayoutChange = (currentLayout: Layout[], allLayouts: Layouts): void => {
```

Comment:
Did you intend to keep these console.log statements?

Reply:
I removed the ones that are not necessary but still left a few in the example code. It helps my partners to understand and consume the control. I will remove them in the future if we no longer find them necessary. 

---
In reply to: [211692210](https://github.com/OfficeDev/office-ui-fabric-react/pull/5899#discussion_r211692210) [](ancestors = 211692210)